.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You can omit this now with the [$schema](https://rejot.dev/docs/spec/manifest/) key in the manifest (also fileMatch is incorrect now) which will be added automatically for new manifests on manifest init to `https://unpkg.com/@rejot-dev/contract@0.0.15/schema.json`